src/reactions/format-reaction.ts

```diff
@@ -0,0 +1,22 @@
+import { ReactionDef } from './reaction-def';
+import { addFormatting } from '../commands/add-formatting';
+
+export const formatReaction: ReactionDef = {
+  emoji: '🤖',
+  description: 'Guesses and formats the message',
+  command: async (reaction) => {
+    try {
+      await reaction.message.reactions.removeAll();
+
+      addFormatting(reaction.message);
+    } catch (error) {
+      // A common issue with this not working correctly, is
+      // if the bot does not have permissions to remove reactions.
+      // Check the README.md file for details on setting the correct permissions
```

Comment:
Should this be changed to the multi-line comment format? To remain consistent with the rest of the code?

```suggestion
      /* A common issue with this not working correctly, is
       * if the bot does not have permissions to remove reactions.
       * Check the README.md file for details on setting the correct permissions
       */

```